### PR TITLE
Feat/swerve

### PR DESCRIPTION
**Pull request by memori034**

### CODE REVIEW

File: src/main/java/frc/robot/RobotType.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2023 FRC 3256
```
Comment (3LucasZ) [inline src/main/java/frc/robot/RobotType.java:1]: We only have one robot type, this isn't necessary.

File: src/main/java/frc/robot/RobotType.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+    ALPHA {
+        @Override
+        public Rotation2d getOffset(int module) {
+            return kAlphaOffsets[module];
+        }
+    };
+
+    public abstract Rotation2d getOffset(int module);
+}
```
Comment (3LucasZ) [inline src/main/java/frc/robot/RobotType.java:31]: please add back TeleopServe

Comment (3LucasZ) [reply src/main/java/frc/robot/RobotType.java:31]: *command

Comment (3LucasZ) [reply src/main/java/frc/robot/RobotType.java:31]: unable to comment on the file itself since the file original was removed, so commenting here

**Review by 3LucasZ — CHANGES_REQUESTED** [review]
Looks to much like a copy paste. Please delete all unnecessary things (LockSwerveX, StatisticsHelper, etc) that are unrelated to the issue you are trying to solve.

**Comment by 3LucasZ** [comment]
Needs a design doc and video of working in simulation before further review. Also made comments above.

File: src/main/java/frc/robot/RobotType.java (hunk 2 above)
Comment (memori034) [reply src/main/java/frc/robot/RobotType.java:31]: can you reexplain please

**Comment by realjoshuau** [comment]
Merge azimuth & rate limits from this

**Pull request closed without merging**